CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # `chainweb-node` Changelog
 
+## UNRELEASED
+
+Changes:
+
+* A new chainwebVersion called fast-development, which is intended to make the devnet experience faster to get started with for testing transactions. See #1627 for more details.
+* Updated from GHC 8.10.7 to GHC 9.6.2. (#1565)
```

Comment:
Not sure if this is relevant for users? Most users probably don't know what "GHC" is.

Drop this or maybe move it down and/or add a bit more context?

Reply:
I can make an "internal changes" section